src/legacy/core_plugins/console/public/src/mappings.js

```diff
@@ -304,7 +304,7 @@ function retrieveAutocompleteInfoFromServer() {
       if (templates) {
         loadTemplates(JSON.parse(templates[0]));
       } else {
-        templates = [];
+        loadTemplates({});
```

Comment:
Noticed another issue here, where we were setting `templates` at the function scope ([L286](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/console/public/src/mappings.js#L286)), not global scope ([L29](https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/console/public/src/mappings.js#L29)).

Reply:
yeah, i like that better! fixed with latest commit.